src/test/java/bo/usfx/luis/fernando/villca/TotalPointsTests.java

```diff
@@ -0,0 +1,30 @@
+package bo.usfx.luis.fernando.villca;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TotalPointsTests {
+    @Test
+    public void testVerifyThatItReturnsTheResultWithAllTheGames() {
+        String[] games = {"1:0", "2:0", "3:0", "4:0", "2:1", "1:3", "1:4", "2:3", "2:4", "3:4"};
+        Assert.assertEquals(15, TotalPoints.points(games));
```

Comment:
Please swap the assertion values:
```java
Assert.assertEquals(Assert.assertEquals(TotalPoints.points(games), 15);
```
Apply this in all your assertions. That is the correct order.